superset/connectors/sqla/models.py

```diff
@@ -1120,7 +1120,9 @@ def is_alias_used_in_orderby(col: ColumnElement) -> bool:
                 col.name = f"{col.name}__"
 
     def get_sqla_row_level_filters(
-        self, template_processor: BaseTemplateProcessor
+        self,
+        template_processor: BaseTemplateProcessor,
+        username: Optional[str] = None,
```

Comment:
nit: `username` is missing in the docstring here and in `security_manager.get_rls_filters`. The main description says it retrieves rls filters for "the current user" - we should perhaps remove that text from both docstrings and state that if `username` is missing, it will fall back to the current user.

Reply:
Ah, good catch and good point, @villebro! Let me fix the docstrings and add tests.